src/utils/popups.js

```diff
@@ -55,15 +57,52 @@ const wcLinkPopupHTML = (title, displayProps) => {
           ${downstreamWCLinks}`;
 };
 
+const fitMapToFeatures = (map, features) => {
```

Comment:
Is this pretty fast? Seems like it could end up being really slow with lots of features. Is this a reason we only return 50 upstream/downstream links, or is the db more of a bottleneck? If the db is the slow part then this is probably irrelevant I guess.

Reply:
>Is this pretty fast?
yeah, I didn't have any problems with it when i was playing around, but maybe i should try it with network throttled and stuff 🤔 

>Is this a reason we only return 50 upstream/downstream links
Nope, 50 was a number I made up for the API - 100 was a bit slow so I thought 50 seemed an okay number to choose  😅 So yeah the db doesn't really affect this function really